src/main/webapp/WEB-INF/view/profiles.jsp

```diff
@@ -1,9 +1,73 @@
+<%@ page import="java.util.List" %>
+
+
+<%--
+  Copyright 2017 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+--%>
+<%@ page import="java.util.List" %>
+<%@ page import="codeu.model.data.User" %>
+<%@ page import="codeu.model.store.basic.UserStore" %>
+
+
+<%
+User user = (User) request.getAttribute("user");
+%>
+
+
+
 <!DOCTYPE html>
 <html>
-    <head>
-        <title>User Profile</title>
-    </head>
-    <body>
-        <h1>This is the user's profile.<h1>
+<head>
+  <title>Register</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+  <script>
```

Comment:
I don't think you need this script on this page? This is only used on the chat page to scroll the chat to the bottom.